requirements.txt

```diff
@@ -1,9 +1,77 @@
-###
-# This file is only here to make sure that something like
-#
-#    pip install -e .
-#
-# works as expected. Requirements can be found in setup.py.
-###
-
-.
+appdirs==1.4.4
+argh==0.26.2
+atomicwrites==1.4.0
+attrs==21.4.0
+awesome-slugify==1.6.5
+Babel==2.9.1
+backports.functools-lru-cache==1.6.4
+backports.ssl-match-hostname==3.7.0.1
+blinker==1.4
+certifi==2021.10.8
+chainmap==1.0.3
+chardet==4.0.0
+click==6.2
+configparser==4.0.2
+contextlib2==0.6.0.post1
+emoji==0.4.5
+enum34==1.1.10
+feedparser==5.2.1
+Flask==0.10.1
+Flask-Assets==0.10
+Flask-Babel==0.9
+Flask-Login==0.2.11
+Flask-Principal==0.3.5
+funcsigs==1.0.2
+future==0.15.2
+futures==3.1.1.post1
+idna==2.10
+importlib-metadata==2.1.2
+importlib-resources==3.3.1
+itsdangerous==1.1.0
+Jinja2==2.8.1
+Markdown==2.6.11
+MarkupSafe==1.1.1
+monotonic==1.3
+more-itertools==5.0.0
+-e git+git@github.com:mrbeam/MrBeamDoc.git@fe6aeb064dee2d5d987465f5dc251c349a4059af#egg=MrBeamDoc
```

Comment:
Not sure about how to include this on the requirements. We should discuss about this. Also I only added the requirements need to run the test locally so far not the requirements needed to run the current version of our system on a rasberrypi. There are some differences I could find by doing pip freeze on a mr beam.

Reply:
The requirements of a octoprint plugin have to be in the setup py plugin_requires section

Reply:
Yes. I added it here so that it's easier to run the tests locally. Not really for the project... But it's true that I should add the dependency to the setup.py for the mrbeamdoc at least.

Reply:
But can this cause problems during install? If it is in multiple locations

Reply:
That was my question for you guys 😓
I think it makes sense to have this somewhere my question is. Does it make trouble to keep it here? should I move it to a different file and leave the requirements file for now? I think we would need to have it in both places or ideally have it here and load it from the setup.py